README.md

```diff
@@ -145,12 +145,36 @@ npx ngrok http 3000
   - Add in Allowed Callback URLs: `http://localhost:3000/api/auth/callback`
   - Add in Allowed logout URLs : `http://localhost:3000`
 
+## A0deploy CLI
+
+Setup the application with ```tenanl.yaml``` file
+**Export data**
+```
+a0deploy export --config_file config.json --format yaml --output_folder .
```

Comment:
What  is `a0deploy`?

Reply:
Ok got it, https://auth0.com/docs/deploy-monitor/deploy-cli-tool/install-and-configure-the-deploy-cli-tool#install-the-deploy-cli-tool, I'll add a note in the readme for this.